SPECS/libreswan/CVE-2024-2357.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/programs/pluto/ikev2_auth.c b/programs/pluto/ikev2_auth.c
+index ad5e3b5ede..fe75d6ea37 100644
+--- a/programs/pluto/ikev2_auth.c
++++ b/programs/pluto/ikev2_auth.c
+@@ -862,7 +862,7 @@ stf_status submit_v2AUTH_generate_initiator_signature(struct ike_sa *ike, struct
+ 						       &signed_octets);
+ 		if (d != NULL) {
+ 			llog_diag(RC_LOG_SERIOUS, ike->sa.logger, &d, "%s", "");
+-			return false;
++			return STF_FATAL;
```

Comment:
Looks like the PR check failed because the patch didn't apply cleanly.

Reply:
Yes, this function itself is not present in libreswan code. I missed that.